customers.go

```diff
@@ -40,6 +40,20 @@ type Customer struct {
 	Errors Errors `json:"errors,omitempty"`
 }
 
+// UpdateCustomer allows updating customer on the update endpoint.
+type UpdateCustomer struct {
+	Name               string      `json:"name"`
```

Comment:
What happens with the other fields, if you want to update just one field?
1. all fields are sent empty to Import API,
2. all fields are removed.
Instead, use pointers and keep `omitempty`.
Test serialization will not send `nil` and will send `""`, if you explicitly set it.